nautobot_golden_config/api/views.py

```diff
@@ -124,12 +128,24 @@ class RemediationSettingViewSet(NautobotModelViewSet):  # pylint:disable=too-man
     filterset_class = filters.RemediationSettingFilterSet
 
 
-class ConfigPlanViewSet(NautobotModelViewSet):  # pylint:disable=too-many-ancestors
+class ConfigPlanViewSet(
+    NautobotAPIVersionMixin, NotesViewSetMixin, ModelViewSetMixin, ReadOnlyModelViewSet
+):  # pylint:disable=too-many-ancestors
```

Comment:
To maintain feature parity let's just pull in the discrete action-based mixins here instead of read-only, since that would disallow `put`, `patch`, `delete`.